programs/ziti-edge-tunnel/package/CPackPackage.cmake

```diff
@@ -96,16 +103,26 @@ install(FILES "${INSTALL_OUT_DIR}/${SYSTEMD_EXECSTARTPRE}"
             WORLD_READ WORLD_EXECUTE
         COMPONENT "${COMPONENT_NAME}")
 
+install(FILES "${INSTALL_OUT_DIR}/${ZITI_PKLA_FILE}"
```

Comment:
Instead of depending on ```polkit-pkla[-compat]```, I create and install ```ziti-edge-tunnel.rules```, which is the correct file for ```PolicyKit``` >= 106. I use ```CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION``` to exclude ```/usr/share/polkit-1/rules.d``` directory from the rpm.